scripts/script.js

```diff
@@ -0,0 +1,77 @@
+/*eslint-disable */
+
+const speakersData = [
+  {
+    photo: './images/one.jpg',
+    name: 'Hellen john',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+  {
+    photo: './images/two.jpg',
+    name: 'Bazu Bazu',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+  {
+    photo: './images/three.jpg',
+    name: 'Kevin Mzito',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+  {
+    photo: './images/four.jpg',
+    name: 'Felix Mtongoi',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+
+  {
+    photo: './images/five.jpg',
+    name: 'Linda Henrys',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+  {
+    photo: './images/six.jpg',
+    name: 'Shruti Shah',
+    title: 'CE0 of African movies Festival',
+    description:
+      'Founder of the African movies festival, with 15 years experience in film making',
+  },
+];
+
+const speakerDetails = document.querySelector('.speaker-area');
+
+speakersData.forEach((speaker) => {
+  speakerDetails.innerHTML += `<article class='speakers-detail'>
+          <img src="${speaker.photo}" class="image" ></img>
```

Comment:
Please make sure all images contain `alt` attributes. It is important for the accessibility and SEO of your web pages.